temporian/core/data/dtype.py

```diff
@@ -40,3 +41,25 @@ def MissingValue(dtype: DType) -> Any:
         return ""
     else:
         raise ValueError(f"Non implemented type {dtype}")
+
+
+def python_type_to_temporian_dtype(python_type: Any) -> DType:
```

Comment:
After merging main, could we move this function to `DType.from_python(python_type)`?